### PR TITLE
add --json to status command to print json

### PR DESCRIPTION
**Pull request by fosterseth**

### CODE REVIEW

**Comment by thenets** [comment]
Related to:
- #411

File: receptorctl/receptorctl/cli.py [hunk 1]
```diff
@@ -49,10 +49,13 @@ def get_rc(ctx):
...
 @cli.command(help="Show the status of the Receptor network.")
 @click.pass_context
-def status(ctx):
+@click.option('--json', 'printjson', help="Print as JSON", is_flag=True)
+def status(ctx, printjson):
     rc = get_rc(ctx)
     status = rc.simple_command("status")
-
+    if printjson:
+        print(status)
```
Comment (thenets) [inline receptorctl/receptorctl/cli.py]: is this right? Shouldn't be something like `json.dumps(status)`

**Review by thenets — APPROVED** [review]

File: receptorctl/receptorctl/cli.py (hunk 1 above)
Comment (fosterseth) [reply receptorctl/receptorctl/cli.py]: at this point in the code, status is already JSON

Comment (fosterseth) [reply receptorctl/receptorctl/cli.py]: I was wrong, it was a python object. calling json.dumps

Comment (thenets) [reply receptorctl/receptorctl/cli.py]: Looks perfect now :)

**Review by thenets — APPROVED** [review]

**Review by shanemcd — APPROVED** [review]

**Pull request merged**